data/db/slr-tokina.xml

```diff
@@ -458,4 +458,16 @@
         </calibration>
     </lens>
 
+    <lens>
+	<!-- Taken with Nikon D600 -->
+	<maker>Tokina</maker>
+	<model>Tokina 17mm f/3.5 RMC II</model>
+	<mount>Nikon F AI</mount>
+	<cropfactor>1.0</cropfactor>
+	<calibration>
+		<distortion model="ptlens" focal="17.0" a="0.01947" b="-0.03345" c="-0.02996" />
+		<tca model="poly3" focal="17.0" vr="1.0000948" vb="1.0002216" />
+	</calibration>
+</lens>
```

Comment:
The indentation has gone off in a few places, most notably here. Could you fix it?

Reply:
yes I'm the uploader on https://github.com/lensfun/lensfun/issues/1432 

indentation/use of tabs instead of spaces fixed